packages/experiments/tslint.json

```diff
@@ -1,8 +1,5 @@
 {
   "extends": [
     "office-ui-fabric-react-tslint"
-  ],
-  "rules": {
-    "no-any": false
-  }
+  ]
```

Comment:
you should leave an empty `rules: {}` in there just for convenience if we need to add something back.